tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -66,6 +66,16 @@ Here is an example of what that can look like:
     linux.release.rv64 = "res://bin/libgdexample.linux.template_release.rv64.so"
 
 
+Libraries are processed in order, so if two sets of feature tags could match
```

Comment:
```suggestion
Entries are matched in order, so if two sets of feature tags could match
```

I'm not sure why, but this would be easier to understand for me.